### PR TITLE
Skip failing graphql test

### PR DESCRIPTION
**Pull request by janisz**
## Description

One graphql test is failing on postgres. Let's skip it and get time for investigation. 
Refs: https://github.com/stackrox/stackrox/commit/b1d101686b7d5a7f33d10c2691403d3ed0bf53d1

## Testing Performed
CI

### CODE REVIEW

**Comment by janisz** [comment]
Current dependencies on/for this PR:
* master
  * **PR #3339** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/3339" target="_blank">Graphite</a>  👈
    * **PR #3340** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/3340" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/3339?utm_source=stack-comment).

File: qa-tests-backend/src/test/groovy/DeploymentEventGraphQLTest.groovy [hunk 1]
```diff
@@ -90,6 +92,7 @@ class DeploymentEventGraphQLTest extends BaseSpecification {
     private final gqlService = new GraphQLService()
 
     @Category(GraphQL)
+    @IgnoreIf({ Env.CI_JOBNAME.contains("postgres") })
```
Comment (dashrews78) [inline qa-tests-backend/src/test/groovy/DeploymentEventGraphQLTest.groovy:95]: I recommend we don't skip it.  I'm waiting on a test to pass and then I will put the fix in review.

https://github.com/stackrox/stackrox/pull/3319

**Review by rhybrillou — APPROVED** [review]

**Comment by janisz** [comment]
Closing as it will be fixed by https://github.com/stackrox/stackrox/pull/3319

**Pull request closed without merging**